lib/mac-token.js

```diff
@@ -3,12 +3,15 @@
 // US Government Users Restricted Rights - Use, duplication or disclosure
 // restricted by GSA ADP Schedule Contract with IBM Corp.
 
+var SG = require('strong-globalize');
+var g = SG();
```

Comment:
You don't need these two lines if there are nothing to translate in that file. You can revert this file.